solr/solrj/src/java/org/apache/solr/client/solrj/SolrRequest.java

```diff
@@ -31,9 +31,15 @@
 import org.apache.solr.common.util.ContentStream;
 
 /**
- * @since solr 1.3
+ * Base class for a SolrJ request to Solr. A common way to send it is via {@link
+ * #process(SolrClient)}.
+ *
+ * <p>Not thread-safe; in fact {@link org.apache.solr.client.solrj.impl.LBSolrClient} will
```

Comment:
😬 

Not an issue for this PR, but man, this is kindof scary to see written out.

Reply:
I'm airing dirty laundry :-)